### ISSUE TITLE
Color contrast too low on table of content's links

### ISSUE THREAD

**Issue by dannycolin** [thunderbird/developer-docs#220]
Second level Links in the table of content have a color contrast of 2.94 but it should be at least 4.5. The issue is because an opacity rule of 0.72 is applied on them.
